internal/acceptance/lakeview_dashboard_test.go

```diff
@@ -0,0 +1,481 @@
+package acceptance
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/databricks/databricks-sdk-go"
+	"github.com/databricks/databricks-sdk-go/service/dashboards"
+	"github.com/databricks/terraform-provider-databricks/common"
+	"github.com/databricks/terraform-provider-databricks/qa"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+type TemplateStruct struct {
+	DisplayName         string
+	WarehouseId         string
+	ParentPath          string
+	SerializedDashboard string
+	FilePath            string
+	EmbedCredentials    string
+}
+
+func MakeTemplate(template TemplateStruct) string {
```

Comment:
This does not need to be public.

Reply:
So where should I put it? Should I also put it in util.go in common?

Reply:
Just make it lowercase:
```
func makeTemplate(...
```